src/ui/server/helpers/summary-lists/your-buyer/trading-history/index.ts

```diff
@@ -0,0 +1,115 @@
+import { YOUR_BUYER_FIELDS as FIELDS } from '../../../../content-strings/fields/insurance';
+import { FORM_TITLES } from '../../../../content-strings/form-titles';
+import INSURANCE_FIELD_IDS from '../../../../constants/field-ids/insurance';
+import { ROUTES } from '../../../../constants';
+import fieldGroupItem from '../../generate-field-group-item';
+import getFieldById from '../../../get-field-by-id';
+import mapYesNoField from '../../../mappings/map-yes-no-field';
+import generateChangeLink from '../../../generate-change-link';
+import { ApplicationBuyerTradingHistory, SummaryListItemData, SummaryListGroupData } from '../../../../../types';
+
+const { YOUR_BUYER: FIELD_IDS } = INSURANCE_FIELD_IDS;
+
+const {
+  YOUR_BUYER: { TRADING_HISTORY: FORM_TITLE },
+} = FORM_TITLES;
+
+const {
+  INSURANCE: {
+    YOUR_BUYER: { TRADED_WITH_BUYER_CHANGE, TRADED_WITH_BUYER_CHECK_AND_CHANGE, TRADING_HISTORY_CHANGE, TRADING_HISTORY_CHECK_AND_CHANGE },
+  },
+} = ROUTES;
```

Comment:
Not a blocker -`INSURANCE_ROUTES`

Also the `FIELD_IDS` could be destructured in a single import line instead of an import and then destructuring